NorimaConsulting/training-and-onboarding
# REC 41/42: Add Recipe List page with delete recipe button functionality

- Added Recipe list page that shows all the recipes in the Hasura database, along with recipe information such as who it was posted by, title, and average rating. Does not included paginator
- Has a delete recipe button that asks for confirmation, it should delete the recipe from the database but currently fixing hook call issue that doesn't allow the delete query to be run.

Testing:
1. Add .env file in root directory with the following variables (see .env.sample file for reference):
```
VITE_HASURA_GRAPHQL_ADMIN_SECRET
VITE_HASURA_GRAPHQL_ENDPOINT="https://winter-2022.hasura.app"
```
2. Go to /webapp and run `pnpm dev`
3. Open `http://localhost:3000/` and click `Recipes` button on the navigation bar. It should route to a page that has all the recipes in their own cards. You should be able to click on each card and get routed to the individual recipe page (for now it should not show anything as Signe's PR handles the individual recipe page functionality
